src/components/TheCanvas.vue

```diff
@@ -86,24 +87,47 @@ export default class TheCanvas extends Vue {
   }
 
   created () {
-    window.addEventListener('resize', (e) => {
-      const currentTarget = e.currentTarget as Window
-      if (currentTarget) {
-        this.$data.stageSize = {
-          width: currentTarget.innerWidth,
-          height: currentTarget.innerHeight
-        }
-      }
-    }, true)
+    window.onload = () => {
+      this.setStageConfig({
+        scale: {
+          x: 1,
+          y: 1
+        },
+        width: 500,
+        height: 500,
+        initialWidth: 500,
+        initialHeight: 500
+      })
```

Comment:
In the future, the sockets need to request and set the stage configurations. Atm its just hardcoded